src/odemis/acq/fastem.py

```diff
@@ -962,49 +963,60 @@ def estimateTiledAcquisitionTime(stream, stage, area):
 
     return acq_time_overview
 
+class OverviewAcquisition(object):
+    """Class to run the acquisition of one overview image (typically one scintillator)."""
+    def __init__(self) -> None:
+        self.sf = None
 
-def _run_overview_acquisition(f, stream, stage, area, live_stream):
-    """
-    Runs the acquisition of one overview image (typically one scintillator).
+    def _cancel_acquisition(self, future):
+        if self.sf:
+            self.sf.cancel()
```

Comment:
A canceller is suppose to return a bool. Please indicate it in the function signature, and in general, you can `return True` (meaning the cancelling succeeded).